### DISCUSSION TITLE
lxqt-admin helpers paths in codes

### DISCUSSION THREAD

**Discussion by hamzehnasajpour** [lxqt/lxqt#2133] [Q&A] [answered]
Is there any specific reason to using absolute path to run `lxqt-admin-time-helper` [here](https://github.com/lxqt/lxqt-admin/blob/77c0944aa891f7fe51e01a687963c5381796a1f4/lxqt-admin-time/nosdtimedatectl.cpp#L132) and run the `lxqt-admin-user-helper` without path [here](https://github.com/lxqt/lxqt-admin/blob/77c0944aa891f7fe51e01a687963c5381796a1f4/lxqt-admin-user/usermanager.cpp#L212)? Both of them are correct, just saying for consistency.

**Comment by palinek** [comment, marked as answer]
I would say, that the lxqt-admin-time-helper invocation is wrong and violates the ability to define installation prefix in build time.

**Reply by hamzehnasajpour** [reply]
Exactly, and this is why I asked. So this patch could be applied

```patch
diff --git a/lxqt-admin-time/nosdtimedatectl.cpp b/lxqt-admin-time/nosdtimedatectl.cpp
index 2149ec9..51ce1f7 100644
--- a/lxqt-admin-time/nosdtimedatectl.cpp
+++ b/lxqt-admin-time/nosdtimedatectl.cpp
@@ -129,7 +129,7 @@ bool NOSDTimeDateCtl::pkexec()
     QProcess process;
     QStringList args;
     args << QSL("--disable-internal-agent")
-        << QSL("/usr/bin/lxqt-admin-time-helper")
+        << QSL("lxqt-admin-time-helper")
         << mHelperArgs;
     process.start(QSL("pkexec"), args);
     process.waitForFinished(-1);
```

**Reply by stefonarch** [reply]
Fixed in https://github.com/lxqt/lxqt-admin/pull/258 thanks!
